packages/modules/product/src/migrations/InitialSetup20240401153642.ts

```diff
@@ -12,88 +12,192 @@ export class InitialSetup20240315083440 extends Migration {
         `alter table "product_option_value" alter column "variant_id" drop not null;`
```

Comment:
nit: we could remove `table_schema = 'public'` from line 6.

Reply:
Should this change be done to all migration files?